tests/input_flows.py

```diff
@@ -381,6 +387,71 @@ def input_flow_tr(self) -> BRGeneratorType:
         self.debug.press_middle()
 
 
+class InputFlowShowXpubQRCode(InputFlowBase):
+    def __init__(self, client: Client, passphrase: bool = False):
+        super().__init__(client)
+        self.passphrase = passphrase
+
+    def input_flow_tt(self) -> BRGeneratorType:
+        if self.passphrase:
+            br = yield
```

Comment:
`br =` not necessary here not two lines below - `br` is not read here

Reply:
[e49b023](https://github.com/trezor/trezor-firmware/pull/3257/commits/e49b0236666409647e5a8663c1d13907f51e594b)